aws/resource_aws_waf_byte_match_set.go

```diff
@@ -24,7 +24,7 @@ func resourceAwsWafByteMatchSet() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-			"byte_match_tuples": &schema.Schema{
```

Comment:
We need to duplicate the entire existing `byte_match_tuples` attribute into the new `byte_match_tuple` attribute. Then in the existing `byte_match_tupes` attribute, add `ConflictsWith` and `Deprecation` fields, e.g.

```go
ConflictsWith: []string{"byte_match_tuple"},
Deprecation: "use `byte_match_tuple` instead",
```